kitchen.yml

```diff
@@ -1,40 +1,129 @@
+# -*- coding: utf-8 -*-
+# vim: ft=yaml
 ---
+# For help on this file's format, see https://kitchen.ci/
 driver:
   name: docker
   use_sudo: false
   privileged: true
-  run_command: /sbin/init
-  provision_command:
-    - curl -L https://bootstrap.saltstack.com | sh -s -- -X     # install Salt and dependencies here to make use of Docker build cache, speeding up tests massively
+  run_command: /lib/systemd/systemd
 
 verifier:
   name: inspec
 
 provisioner:
   name: salt_solo
-  salt_version: latest
   log_level: info
+  salt_install: none
   require_chef: false
   formula: vault
+  salt_copy_filter:
+    - .kitchen
+    - .git
 
 platforms:
-  - name: ubuntu-16.04
+  - name: amazonlinux
     driver:
-      name: docker
+      image: amazonlinux:1
+      platform: rhel
+      run_command: /sbin/init
       provision_command:
-        - apt-get install -y locales net-tools && locale-gen en_US.UTF-8
-        - curl -L https://bootstrap.saltstack.com | sh -s -- -X
-  - name: centos-7
+        - curl -L https://bootstrap.saltstack.com | sh -s -- -X     # install latest stable Salt
```

Comment:
Does `amazonlinux` still work for local testing?  If not, it's better to remove this block.

Reply:
It does still work, and is useful as it tests the `upstart` init support

Reply:
That's fine, thanks for explaining that.